Assets/Plugins/UniRx/Scripts/Operators/ContinueWith.cs

```diff
@@ -56,9 +56,20 @@ public override void OnCompleted()
             {
                 if (seenValue)
                 {
-                    var v = parent.selector(lastValue);
-                    // dispose source subscription
-                    serialDisposable.Disposable = v.Subscribe(observer);
+                    try
+	                {
+		                var v = parent.selector(lastValue);
+		                // dispose source subscription
+		                serialDisposable.Disposable = v.Subscribe(observer);
+	                }
+	                catch (Exception error)
+	                {
+		                OnError(error);
+	                }
+	                finally
+	                {
+		                Dispose();
```

Comment:
`Dispose()` is always called immediately after `serialDisposable.Disposable = v.Subscribe(observer);`
This will cancel the subscription if it's not finished immediately.
I think the finally block is totally not needed.